RestApi/Sepes.RestApi/Controllers/ValuesController.cs

```diff
@@ -21,6 +25,8 @@ public ActionResult<IEnumerable<string>> Get()
         [HttpGet("{id}")]
         public ActionResult<string> Get(int id)
         {
+            TelemetryClient telclient = new TelemetryClient();
```

Comment:
Hmm this seems wrong to me. Should use dependency injection so that we can stop loging in production. But after googling a bit it seems like a correct enough way. 
Could you verify that we can unit test actions without sending logs and metrics to Azure.
My guess we need make sure that ApplicationInsights is configured to a disabled state when unit tests are running.

Ether way I would recommend moving it to a private readonly variable. And have it created when the controller get created.

Reply:
Once actual functionality is added I will also take unit testing into consideration. Currently there are no functions to set up "real" tests on.